image: work_map3.ppm
resolution: 0.05
origin: [-7.15, -3.8, .0]
negate: 0
occupied_thresh: 0.65
free_thresh: 0.2

radius: .8
origin_x: -7.15
origin_y: -3.8
map_scale: 3
